examples/guestbook/README.md

```diff
@@ -65,20 +70,24 @@ redis-master        dockerfile/redis    kubernetes-minion-2.c.myproject.internal
 If you ssh to that machine, you can run `docker ps` to see the actual pod:
 
 ```shell
-me@workstation$ gcloud compute ssh --zone us-central1-b kubernetes-minion-2
```

Comment:
Please also mention this command as you've mentioned GCE related commands elsewhere.